### ISSUE TITLE
Add Open Orders tab to inventory summary popup with ability to unreserve

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1046]
_Imported from trac ticket 1046.
Created by **rneville**
Opened in trac: Aug 30, 2011, 2:27:53 PM
Last modified in trac: Oct 25, 2011, 3:51:38 PM_
***
probably should only be non-voided lines - still applies, yes
probably should default to be only open, untrucked SAL and RES, by definition yes that's probably all that will show in the grid anyways...

Two birds with one stone.

This is basically what OOSL does anyway and also solves a potential future problem with respect to store based security issues.

**Comment by kbower** [comment]
_**rneville** trac comment on Sep 28, 2011, 4:14:46 PM:_

Eldo wants this for OOSL equivalence

**Comment by kbower** [comment]
_**kbower** trac comment on Sep 29, 2011, 6:59:01 AM:_

This is blocked by being able to open more than one order, so this one's some effort.

**Comment by kbower** [comment]
_**rneville** trac comment on Sep 29, 2011, 7:16:14 AM:_

?? I don't follow? There's no need to open more than one customer order at a time here. This is a link on order line edit that jumps back to order query and runs a query for open orders with same product. Then from the order query screen you can drill-down as always to any order you wish. The only caveat here is handling any unsaved changes to the current order before leaving it. Unless I'm seriously missing something this should be relatively simple.

**Comment by kbower** [comment]
_**kbower** trac comment on Sep 29, 2011, 7:26:51 AM:_

You're not missing anything, that's the point.  The intent of this functionality is they are working with ticket "ABC" and want to find other orders for manual unreserving with the intent of coming back to ticket "ABC" so they can finish working with it.  We can give them what you propose as a start, but it is only half the intention so once we deliver it the first comment will be "I need to save my order and then look it up again when I'm done?"

Of course, whatever they change when they leave the screen probably means the current order needs to be refreshed from the database anyway!  So maybe we can't offer my solution practically anyway.

**Comment by kbower** [comment]
_**rneville** trac comment on Sep 29, 2011, 7:34:32 AM:_

Okay, I see what you're saying. I would propose solving that a different way perhaps....maybe there needs to be a universal "recent stuff" feature that works somewhat like the back button on the browser (or maybe we just fix the back button) that would allow them to jump back...either way, I see that as related to this, but a separate issue.

**Comment by kbower** [comment]
_**rneville** trac comment on Oct 25, 2011, 3:51:38 PM:_

fixed in [[1884]](../commit/440135dcc8d487f73e92b249df36dde9c1cef915)

**Comment by kbower** [comment]
_**rneville** trac status change on Oct 25, 2011, 3:51:38 PM:_ **closed**
